pkg/storaged/stratis-panel.jsx

```diff
@@ -129,13 +129,13 @@ export function create_stratis_pool(client) {
                              },
                              spaces: get_available_spaces(client)
                          }),
-            CheckBoxes("encrypt_pass", client.features.stratis_crypto_binding ? _("Encryption") : "",
+            CheckBoxes("encrypt_pass", client.features.stratis_crypto_binding ? _("Options") : "",
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18867-20230821-100016-7cac847f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)